### ISSUE TITLE
drag does not work in safari

### ISSUE THREAD

**Issue by e03ccd3df65a2d128247** [sveltejs/svelte-repl#110]
Repl has some problem handling `drag` in safari. It works well in chrome.

Here is the [repl](https://svelte.dev/repl/3f9725789ccc4409907446789d5befb0?version=3.22.1)

Drag the red box to the right box and drop. In chrome, it works as expected, but in safari, it does not. It seems something is wrong in safari.

BTW: Svelte REPL is the best repl I ever used❤️. Thanks!

**Comment by multics** [comment]
Yes, I have encountered the similar issue.

**Comment by Conduitry** [comment]
Is this specific to the REPL? Does this work in Safari when downloading the project and running `npm run dev` yourself?

**Comment by multics** [comment]
It's specific to REPL. As a standalone project, it works well in Safari.

**Comment by e03ccd3df65a2d128247** [comment]
> Is this specific to the REPL? Does this work in Safari when downloading the project and running `npm run dev` yourself?

Yes. It is specific to the REPL. There is not problem in Safari.
